### PR TITLE
Ft signup form

### PR DESCRIPTION
**Pull request by jrgrantham**
# Description

Created register form for buyers and schools.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change status

- [x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

Recently pulled from Development.
Used the form and logged the results in the console.
Testing is set up and working.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation, if relevant
- [x] I have run the app using my feature and ensured that no functionality is broken
- [x] My changes generate no new warnings
- [ ] There are no merge conflicts

### CODE REVIEW

**Comment by ruairidhflint** [comment]
Could we have a breakpoint around 450px? Below that point and there is a clash between the buyer/school tab and the Register title and a horizontal bar will soon appear due to the width. 

Design looks super clean, love it.

File: src/Components/RegisterForm.js [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+        axios
+          .post(registerApi, newUser)
+          .then(res => {
+            actions.resetForm();
+            actions.setSubmitting(false);
+            props.loadingFinished();
+            props.history.push("/confirmation");
+          })
+          .catch(err => {
+            debugger;
```
Comment (iamcristos) [inline src/Components/RegisterForm.js]: you left debugger in here man

File: src/Components/RegisterForm.js [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+          password: values.password
+        };
+        actions.setSubmitting(true);
+        axios
+          .post(registerApi, newUser)
+          .then(res => {
+            actions.resetForm();
+            actions.setSubmitting(false);
+            props.loadingFinished();
+            props.history.push("/confirmation");
```
Comment (iamcristos) [inline src/Components/RegisterForm.js]: this doesn't seem to be the page where the user sees check your email

File: src/Components/RegisterForm.js (hunk 1 above)
Comment (iamcristos) [reply src/Components/RegisterForm.js]: error is not also handled

**Pull request merged**